docs/components/modeler/desktop-modeler/process-applications/create-process-application.md

```diff
@@ -0,0 +1,13 @@
+---
+id: create-process-application
+title: Create a process application
+description: Learn how to create a process application in Desktop Modeler.
+---
+
+To create a process application, take the following steps:
```

Comment:
You can also create a process application by adding an empty `.process-application` file in the root, right?

Reply:
Yes, how you create the file doesn't matter.

Reply:
However, if you create the file outside of the modeler you might have to restart it. When creating the process application through the modeler the folder is explicitly added as a root and opening any file the process applicaton file will be found. However, when a file is already open and a process application file is added outside the modeler that change isn't picked up until you restart the modeler or close and open the file again.

Reply:
It will be picked up when I re-open the file, right?

Reply:
Yes. It only isn't picked up for a file that you already have open when creating the `.process-application` file through non-modeler means.